.github/workflows/lint-codeql.yaml

```diff
@@ -1,7 +1,9 @@
 name: codeql
 
 on:
-  pull_request: {}
+  pull_request:
+    branches:
+      - master
```

Comment:
Don't we also want it to run for backport PRs?

Reply:
It looks like the release branches have their own version of this workflow with `on.push.branches` set to the respective branch. Since this is triggered `on.pull_request` (not `on.pull_request_event`), I think the workflow from the corresponding PR branch is run. Or did I mix something up there?

Reply:
> Since this is triggered on.pull_request (not on.pull_request_event), I think the workflow from the corresponding PR branch is run.

Forgot about that :facepalm: